src/cloud/release-notes/ece-release-notes.md

```diff
@@ -0,0 +1,564 @@
+---
+group: cloud-guide
+title: ECE-Tools release notes
```

Comment:
should this be in Docker release review?

Reply:
It's not a pristine branch.  So, if it's not Docker, you have a chance to review when I submit the Cloud-2002.1.0 PR.